tests/fixtures/addon/ember-cli-build-at-root/ember-cli-build.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Can we test this with our custom fixturify-project test helper (instead of making more static fixture files)?